docs/development.md

```diff
@@ -54,7 +54,7 @@ This repository contains the main code, but two more repositories are included d
 If you’re familiar with TypeScript, the [Theia IDE](https://theia-ide.org/), and if you want to contribute to the
 project, you should be able to build the Arduino IDE locally.
 Please refer to the [Theia IDE prerequisites](https://github.com/eclipse-theia/theia/blob/master/doc/Developing.md#prerequisites) documentation for the setup instructions.
-> **Note**: Node.js 14 must be used instead of the version 12 recommended at the link above.
+> **Note**: Node.js 16 must be used instead of the version 14 recommended at the link above. `arm64` arch is not recognized on 14. If you want to work build IDE2 on ARM, Node.js 16 is a must.
```

Comment:
```suggestion
> **Note**: Node.js 16 must be used instead of the version 14 recommended at the link above.
```

The Node.js requirement is already specified in the sentence and ensured via the `engines.node` field of `package.json` so I feel this part only adds unnecessary verbosity.